### PR TITLE
doc: Archive v0.16.1.1 release notes, draft v0.17 release notes

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -40,7 +43,7 @@ Downgrading to a version older than 0.14.0.3 is no longer supported due to
 changes in the "evodb" database format. If you need to use an older version,
 you must either reindex or re-sync the whole chain.
 
-### Downgrade of masternodes to < 0.16
+### Downgrade of masternodes to < 0.17
 
 Starting with this release, masternodes will verify the protocol version of other
```
Comment (thephez) [inline doc/release-notes.md:48]: This paragraph was for 0.16 and doesn't need to be here, correct?

File: doc/release-notes.md [hunk 2]
```diff
@@ -49,27 +52,223 @@ so downgrading is not recommended.
...
+Wallet changes
 --------------
+Upgrading a non-Hierarchical Deterministic (HD) wallet to an HD wallet is now
+possible. Previously new HD wallets could be created, but non-HD wallets could
+not be upgraded. This update will enable existing non-HD wallets to upgrade to
+take advantage of HD wallet features. Upgrades can be done via either the debug
+console or command line and a new backup must be made when this upgrade is
+performed.
+
+Users can now load and unload wallet dynamically via RPC and GUI console.
```
Comment (thephez) [inline doc/release-notes.md:120]:
```suggestion
Users can now load and unload wallets dynamically via RPC and GUI console.
```

File: doc/release-notes.md [hunk 3]
```diff
@@ -49,27 +52,223 @@ so downgrading is not recommended.
...
+It's now possible to have multiple wallets loaded at the asme time, however only
```
Comment (thephez) [inline doc/release-notes.md:121]:
```suggestion
It's now possible to have multiple wallets loaded at the same time, however only
```

File: doc/release-notes.md [hunk 4]
```diff
@@ -49,27 +52,223 @@ so downgrading is not recommended.
...
+one of them is going to be active and receive updates or be used for sending
+trasactions at a specific point in time.
+
+Also, no wallet restart is required on wallet encryption anymore.
```
Comment (thephez) [inline doc/release-notes.md:125]: Maybe re-word to something like this?
```suggestion
Also, enabling wallet encryption no longer requires a wallet restart.
```

File: doc/release-notes.md [hunk 5]
```diff
@@ -49,27 +52,223 @@ so downgrading is not recommended.
...
+
+Packages libX11, libXext, xextproto and xtrans are no longer used.
+
+Minimum supported macOS version was bumped to 10.10.
+
+RPC changes
+-----------
+There are six new RPC commands which are Dash specific and seven new RPC
+commands introduced through Bitcoin backports. One previously deprecated RPC,
+estimatefee, was removed and several RPCs have been deprecated.
```
Comment (thephez) [inline doc/release-notes.md:166]: Maybe?
```suggestion
`estimatefee`, was removed and several RPCs have been deprecated.
```

File: doc/release-notes.md [hunk 6]
```diff
@@ -49,27 +52,223 @@ so downgrading is not recommended.
...
+- `getreceivedbyaccount`
+- `listaccounts`
+- `listreceivedbyaccount`
+- `masternode current`
+- `masternode winner`
+- `move`
+- `sendfrom`
+- `setaccount`
+
+`protx register` and `protx register_fund` rpcs now accept additional param
```
Comment (thephez) [inline doc/release-notes.md:202]:
```suggestion
`protx register` and `protx register_fund` RPCs now accept an additional `submit` param
```

File: doc/release-notes.md [hunk 7]
```diff
@@ -49,27 +52,223 @@ so downgrading is not recommended.
...
+`submit` which allows to produce and print ProRegTx-es without relaying them
```
Comment (thephez) [inline doc/release-notes.md:203]:
```suggestion
which allows producing and printing ProRegTx-es without relaying them
```

File: doc/release-notes.md [hunk 8]
```diff
@@ -49,27 +52,223 @@ so downgrading is not recommended.
...
+to the network.
+
+Also, please note that all mixing-related RPCs have been renamed to replace
+“PrivateSend” with “CoinJoin” (e.g. `setprivatesendrounds` -> `setcoinjoinrounds`).
+
+Please check `help <command>` in rpc for more information.
```
Comment (thephez) [inline doc/release-notes.md:209]: Original wording a little confusing. Maybe something like:
```suggestion
Please check `help <command>` for more detailed information on specific RPCs.
```

File: doc/release-notes.md [hunk 9]
```diff
@@ -49,27 +52,223 @@ so downgrading is not recommended.
...
+--------------------
+Changes in existing cmd-line options:
+
+New cmd-line options:
+- `-dip8params`
+- `-llmq-data-recovery`
+- `-llmq-qvvec-sync`
+- `-llmqinstantsend`
+- `-platform-user`
+- `-statsenabled`
```
Comment (thephez) [inline doc/release-notes.md:221]: Is this the first release that includes the statoshi stuff? Might be worth mentioning in the release notes.

**Review by thephez — COMMENTED** [review]
Looks good for the most part. I had some suggestions and questions.

File: doc/release-notes.md [hunk 10]
```diff
@@ -45,8 +45,8 @@ you must either reindex or re-sync the whole chain.
 
 ### Downgrade of masternodes to < 0.17
 
-Starting with this release, masternodes will verify the protocol version of other
-masternodes. This will result in PoSe punishment/banning for outdated masternodes,
+Starting with 0.16 release, masternodes verify the protocol version of other
```
Comment (thephez) [inline doc/release-notes.md:48]:
```suggestion
Starting with the 0.16 release, masternodes verify the protocol version of other
```

**Review by thephez — DISMISSED** [review]
:+1:

File: doc/release-notes.md [hunk 11]
```diff
@@ -40,36 +43,241 @@ Downgrading to a version older than 0.14.0.3 is no longer supported due to
...
+- `getzmqnotifications`
+- `gobject list-prepared`
+- `listlabels`
+- `masternode payments`
+- `quorum getdata`
+- `quorum verify`
+- `signrawtransactionwithkey`
+- `signrawtransactionwithwallet`
+- `unloadwallet`
+- `verifyislock`
```
Comment (xdustinface) [inline doc/release-notes.md:190]:
```suggestion
- `verifychainlock`
- `verifyislock`
```

File: doc/release-notes.md [hunk 12]
```diff
@@ -40,36 +43,241 @@ Downgrading to a version older than 0.14.0.3 is no longer supported due to
...
+- `getreceivedbyaccount`
+- `listaccounts`
+- `listreceivedbyaccount`
+- `masternode current`
+- `masternode winner`
+- `move`
+- `sendfrom`
+- `setaccount`
+
+`protx register` and `protx register_fund` RPCs now accept an additional `submit` param
```
Comment (xdustinface) [inline doc/release-notes.md:211]:
```suggestion
`protx register` and `protx register_fund` RPCs now accept an additional `submit` parameter
```

File: doc/release-notes.md [hunk 13]
```diff
@@ -40,36 +43,241 @@ Downgrading to a version older than 0.14.0.3 is no longer supported due to
...
+- qt 5.9.6
+- zeromq 4.2.3
+
+Packages libX11, libXext, xextproto and xtrans are no longer used.
+
+Minimum supported macOS version was bumped to 10.10.
+
+RPC changes
+-----------
+There are six new RPC commands which are Dash specific and seven new RPC
```
Comment (xdustinface) [inline doc/release-notes.md:173]:
```suggestion
There are seven new RPC commands which are Dash specific and seven new RPC
```

**Review by xdustinface — CHANGES_REQUESTED** [review]

File: doc/release-notes.md [hunk 14]
```diff
@@ -1,11 +1,14 @@
...
 
 Release is now available from:
 
   <https://www.dash.org/downloads/#wallets>
 
-This is a new hotfix release.
+This is a new major version release, bringing new features, various bugfixes
+and other improvements.
+
+This release is mandatory for all nodes.
```
Comment (PastaPastaPasta) [inline doc/release-notes.md:11]: We should expound further, detailing that this release includes a BIP9 style hard fork that will begin signalling on July 1st, and that any nodes that do not upgrade by the time the HF is activated will diverge from the rest of the network

**Review by PastaPastaPasta — CHANGES_REQUESTED** [review]

File: doc/release-notes.md [hunk 15]
```diff
@@ -1,11 +1,14 @@
-Dash Core version 0.16.1.1
-==========================
+Dash Core version 0.17
```
Comment (PastaPastaPasta) [inline doc/release-notes.md:1]: add .0.0?

**Review by PastaPastaPasta — CHANGES_REQUESTED** [review]

File: doc/release-notes.md (hunk 15 above)
Comment (UdjinM6) [reply doc/release-notes.md]: done, see 9cb7e57

**Review by xdustinface — APPROVED** [review]
utACK

File: doc/release-notes.md (hunk 14 above)
Comment (PastaPastaPasta) [reply doc/release-notes.md:11]: bump?

Comment (UdjinM6) [reply doc/release-notes.md:11]: pls see 4d1c0b7

File: doc/release-notes.md [hunk 16]
```diff
@@ -40,36 +43,247 @@ Downgrading to a version older than 0.14.0.3 is no longer supported due to
...
+Statoshi backport
+------------------
+This version includes a [backport](https://github.com/dashpay/dash/pull/2515)
+of [Statoshi functionality](https://github.com/jlopp/statoshi) which allows
+nodes to emit metrics to a StatsD instance. This can help node operators to
+learn more about node performance and network state in general. We added
+several command line options to give node operators more control, see options
+starting with `-stats` prefix.
+
+PrivateSend rename
```
Comment (PastaPastaPasta) [inline doc/release-notes.md:153]: maybe call it a debranding?

**Review by PastaPastaPasta — APPROVED** [review]
besides one minor thought, LGTM

File: doc/release-notes.md (hunk 16 above)
Comment (UdjinM6) [reply doc/release-notes.md:153]: that's from https://blog.dash.org/updated-product-brief-dashcore-release-v0-17-339f739b5cfa :)

**Pull request merged**